model/src/w3updtmd.F90

```diff
@@ -1803,6 +1807,9 @@ SUBROUTINE W3UICE ( VA )
                                   ICEI(IX,IY), 'ICE (NEW)'
 #endif
                 VA(:,JSEA) = 0.
+#if defined(W3_UWM) || defined(CESMCOUPLED)
+               charn(jsea) = aalpha
```

Comment:
Curious why this and the below update was added?  I'm wondering if this should really be behind switches or just actual additions to the code?

Reply:
I'm not clear on what you're referring to as a switch here? In my switch_meshcap file I have an item "UWM" to add this line only for UWM or CESM. How should I have added it?

The line was added as part of testing for restart repro.

Reply:
I was just trying to avoid modification of ww3 code for anything other than these two configurations.

Reply:
I ran some tests and we do get differences, which is not unaccepted necessarily.  However, we it's taking me longer than expected to make sure there's are the diffs we want.  For now we can keep this behind the switches here, but I think we can make this more general in the very near future.

Reply:
Thanks for testing; I'm behind on getting this and the UWM PR opened so I haven't addressed any comments yet. I needed a fix in the cap code and I've had trouble w/ the c192 test resources. I think I've resolved the issues and I'm testing hera and orion now.

Reply:
Is the 1/4 deg grid going to change at all?  If not, if it could be copied to hera (or another machine other than Cheyenne) I could in parallel start creating ICs.  Apologies if that's already been done and I've missed that.

Reply:
The follow up for this issue is here: https://github.com/NOAA-EMC/WW3/issues/677